src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -223,9 +221,7 @@ public AgentSettings LoadSettings()
                 agentName = consoleWizard.ReadValue(CliArgs.Agent,
                                                 StringUtil.Loc("AgentName"),
                                                 false,
-                                                // TODO: coreCLR doesn't expose till very recently (Jan 15)
-                                                // Environment.MachineName,
-                                                "myagent",
+                                                Environment.MachineName,
```

Comment:
I've heard tales of it being null before on a customer's machine. Just something to be aware of to make sure you don't null ref anywhere if it happens to be null.